common.py

```diff
@@ -61,10 +61,9 @@ def load_from_cache(layer):
 
 def save_to_cache(image):
     '''Given an image object, check against the cache to see if a layer id
-    exists. If no then save the layer and package list to the cache'''
```

Comment:
We're not doing any checks whether a layer id exists in the cache at all. So "Given an image object, save all the layers to the cache"